dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/preheat/mappers/CategoryOptionMapper.java

```diff
@@ -28,13 +28,25 @@
 package org.hisp.dhis.tracker.preheat.mappers;
 
 import org.hisp.dhis.category.CategoryOption;
+import org.mapstruct.BeanMapping;
 import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
 import org.mapstruct.factory.Mappers;
 
 @Mapper( uses = DebugMapper.class )
 public interface CategoryOptionMapper extends PreheatMapper<CategoryOption>
 {
     CategoryOptionMapper INSTANCE = Mappers.getMapper( CategoryOptionMapper.class );
 
+    @BeanMapping( ignoreByDefault = true )
```

Comment:
@enricocolasante would be in favor of adding this explicit mapping?

The reason why I originally added it was to prevent the lazy init exception we got from mapping the collection fields. We learned that this was due to the `manager.getDefaults()` not really returning a detached entity. Its collection fields were still hibernate proxies, and since these were cached in another thread the session was closed => lazy init exception.

Now that I wrote a specific `DefaultsSupplier` that uses the manager to fetch the entities, this will not occur.
This is why I am asking. So this change is not needed to fix defaults. It would only align with being explicit which we do in most places. We could of course make this change its own PR at some point. In all remaining mappers.

Reply:
I removed the change since it is not needed to fix defaults with respect to idSchemes. Forgetting to map a field can lead to another bug. So if we want to use explicit mapping everywhere, we should do so in another task.